src/components/Spawn/IconTooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import Tooltip from "@leafygreen-ui/tooltip";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+
+interface IconTooltipProps {
+  tooltipText: string;
+  iconType: string;
+  color: string;
+  shade: string;
```

Comment:
I think this would make more sense for it to accept any color as a prop as opposed to relying on keys of uiColors.